Open Judge System/Workers/OJS.Workers.Common/Helpers/JavaCodePreprocessorHelper.cs

```diff
@@ -7,27 +7,43 @@
     public static class JavaCodePreprocessorHelper
     {
         private const string PackageNameRegEx = @"\bpackage\s+[a-zA-Z_][a-zA-Z_.0-9]{0,150}\s*;";
-        private const string ClassNameRegEx = @"public\s+class\s+([a-zA-Z_][a-zA-Z_0-9]{0,150})\s*{";
+        private const string PublicClassNameRegEx = @"public\s+class\s+([a-zA-Z_][a-zA-Z_0-9]{0,150})\s*{";
```

Comment:
IMO it may be good to have comments for each regex - what it matches, for better readability, @KristianMariyanov ?